rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2363,6 +2363,40 @@ void voteFederationChange_commitEmptyFederation_returnsInsufficientMembersRespon
 
         }
 
+        @Test
+        void voteFederationChange_addFederatorPublicKeyWithLessThanMofNVotes_returnsSuccessfulResponseCodeAndPendingFederationSizeZero() {
+
+            // Arrange
+
+            List<BtcECKey> fedKeys = BitcoinTestUtils.getBtcEcKeysFromSeeds(
+                new String[]{"key1"}, true
+            );
+
+            BtcECKey expectedBtcECKey = fedKeys.get(0);
+
+            Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, FederationChangeCaller.FIRST_AUTHORIZED.getRskAddress());
+            Transaction tx2 = TransactionUtils.getTransactionFromCaller(signatureCache, FederationChangeCaller.SECOND_AUTHORIZED.getRskAddress());
+
+            ABICallSpec createFederationAbiCallSpec = new ABICallSpec(FederationChangeFunction.CREATE.getKey(), new byte[][]{});
+
+            // Voting with m of n authorizers to create the pending federation
+            federationSupport.voteFederationChange(tx, createFederationAbiCallSpec, signatureCache, bridgeEventLogger);
+            federationSupport.voteFederationChange(tx2, createFederationAbiCallSpec, signatureCache, bridgeEventLogger);
+
+            ABICallSpec addFederationAbiCallSpec = new ABICallSpec(FederationChangeFunction.ADD.getKey(), new byte[][]{expectedBtcECKey.getPubKey()});
```

Comment:
this name is not correct, you are not adding a federation but a member, should be `addFederatorBtcPublicKeyAbiCallSpec`

Reply:
Updated.